File: easy/1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        int count=0;
        vector<int> less = {};
        for (int i = 0; i < size(nums); i++){
            for (int j = 0; j < size(nums); j++){
                if (nums[j] < nums[i])
                    count++;
            }
            less.push_back(count);
            count = 0;
        }

        return less;
    }
};

File: easy/1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        srted = sorted(nums)
        less=[0]*len(nums)

        unique = {}
        for i, n in enumerate(srted):
            if n not in unique:
                unique[n] = i

        for i in range(len(nums)):
            less[i] = unique[nums[i]]

        return less